### PR TITLE
- Add the HA-SAP supportutils plugin to all SAP images

### PR DESCRIPTION
**Pull request by rjschwei**

### CODE REVIEW

File: images/cross-cloud/sles-sap/azure-li/profiles.yaml [hunk 1]
```diff
@@ -3,6 +3,7 @@ profiles:
     include:
       - base/common
       - base/sle
+      - products/sap
       - products/sap/azure-baremetal
```
Comment (jgleissner) [inline images/cross-cloud/sles-sap/azure-li/profiles.yaml:7]: Superfluous (applies to all the changes in `images/`).

**Review by jgleissner — CHANGES_REQUESTED** [review]

File: data/products/sap/packages.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+packages:
+  image:
+    sap-support-config-plugin:
+      - supportutils-plugin-ha-sap
```
[the remark this answers is not in the document]
Comment (rjschwei) [reply data/products/sap/packages.yaml:4]: But that means that for everything that is common between the products we have to duplicate the information :(

[the remark this answers is not in the document]
Comment (jgleissner) [reply data/products/sap/packages.yaml:4]: This is correct. We decided on keeping the code streams separate to keep it simpler and more resistant to accidental change, but it does result in some duplication. We can reconsider, of course.

**Comment by rjschwei** [comment]
It's already there :)

**Pull request closed without merging**